pyuvsim/tests/test_simsetup.py

```diff
@@ -294,12 +319,24 @@ def test_mock_catalogs():
                      'triangle': 'mock_triangle_2458098.27471.txt',
                      'zenith': 'mock_zenith_2458098.27471.txt'}
```

Comment:
now that you have the seed, you could test the random array in the same way, specifying the seed to be the same as in the saved file.

Reply:
I've added that in the latest commit.